### PR TITLE
feat(cdk): rows copy feature and reworks

### PR DESCRIPTION
**Pull request by Schirbak**
## PR Checklist

Please check if your PR fulfills the following requirements:

-   [x] The commit message follows our guidelines: https://github.com/ngxs/store/blob/master/CONTRIBUTING.md#commit
-   [x] Tests for the changes have been added (for bug fixes / features)
-   [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: packages/cdk/integration/tests/jest/table-utils/table-clipboard.spec.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    });
+
+    afterEach((): void => {
+        copySpy.mockReset();
+    });
+
+    it('should correctly convert to plain html', async () => {
+        await tableClipboard.generateTableAndCopy({ entries: dataset });
+        await new Promise((resolve) => setTimeout(resolve));
+        const blob = copySpy.mock.calls[0][0][0].types['text/html'];
```
Comment (splincode) [inline packages/cdk/integration/tests/jest/table-utils/table-clipboard.spec.ts:65]: `[0][0][0]` )))

Comment (splincode) [reply packages/cdk/integration/tests/jest/table-utils/table-clipboard.spec.ts:65]: if later you need refactor, you forgot about what is it

Comment (Schirbak) [reply packages/cdk/integration/tests/jest/table-utils/table-clipboard.spec.ts:65]: well well wellll

**Review by splincode — APPROVED** [review]

**Comment by Schirbak** [comment]
:pepelove:

**Pull request merged**